tests/models/score_network/test_score_network.py

```diff
@@ -45,57 +64,103 @@ def assert_parameters_are_the_same(parameters1: dataclass, parameters2: dataclas
 
 
 @pytest.mark.parametrize("spatial_dimension", [2, 3])
+@pytest.mark.parametrize("num_atom_types", [3])
 class TestScoreNetworkCheck:
 
     @pytest.fixture(scope="class", autouse=True)
     def set_random_seed(self):
         torch.manual_seed(123)
 
     @pytest.fixture()
-    def base_score_network(self, spatial_dimension):
+    def base_score_network(self, spatial_dimension, num_atom_types):
         return ScoreNetwork(
             ScoreNetworkParameters(
-                architecture="dummy", spatial_dimension=spatial_dimension
+                architecture="dummy",
+                spatial_dimension=spatial_dimension,
+                num_atom_types=num_atom_types,
             )
         )
 
     @pytest.fixture()
-    def good_batch(self, spatial_dimension):
+    def good_batch(self, spatial_dimension, num_atom_types):
         batch_size = 16
```

Comment:
define a variable `number_of_atoms = 8` to avoid magic number in the next lines.